Formula/ddcctl.rb

```diff
@@ -0,0 +1,32 @@
+class Ddcctl < Formula
+  desc "DDC monitor controls (brightness) for Mac OSX command-line"
+  homepage "https://github.com/kfix/ddcctl"
+  url "https://github.com/kfix/ddcctl/archive/02dcf6c9ae77783f50a9948a829dee6865131ef6.tar.gz"
+  version "02dcf6c9ae77783f50a9948a829dee6865131ef6"
+  sha256 "e591e14c06b3b8cab2e0b092f8337b60dd6d472c1e2e981352721851fab13085"
+  head "https://github.com/kfix/ddcctl"
+
+  def install
+    if gpu_vendors.find("amd")
```

Comment:
This doesn't work as it gets compiled on CI and run on the user's machine. The software needs to allow all configurations or this can't be in homebrew-core.